easybuild/easyblocks/c/cuda.py

```diff
@@ -114,3 +148,12 @@ def make_module_req_guess(self):
         })
 
         return guesses
+    
+    def make_module_extra(self):
+        """Specify CUDA_HOME """
+
+        txt = super(EB_CUDA, self).make_module_extra()
+
+        txt += self.module_generator.set_environment('CUDA_HOME', self.installdir)
```

Comment:
Bug that causes the issue with `$CUDA_HOME` is fixed in https://github.com/hpcugent/easybuild-framework/pull/1519